sdk/identity/identity/src/tokenCache/InMemoryPersistence.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as msalCommon from "@azure/msal-common";
+import { CachePlugin } from "./types";
+
+/**
+ * To allow silent authentications on the same credential, we provide a very simple in memory token cache.
+ * It can't be used to re-use the account information returned from the authenticate() method.
+ * @internal
+ */
+export class InMemoryPersistence {
+  private inMemoryCache: string = "";
+
+  async register(): Promise<CachePlugin> {
```

Comment:
Is it possible that this would get registered to more than one MSAL client? If so I believe they will stomp each others updates without any synchronization or merging logic.

Reply:
Only one per client.